portability-api/src/test/java/org/dataportabilityproject/webapp/StartCopyHandlerTest.java

```diff
@@ -0,0 +1,99 @@
+package org.dataportabilityproject.webapp;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.apache.axis.transport.http.HTTPConstants.HEADER_COOKIE;
+
+import com.sun.net.httpserver.Headers;
+import org.dataportabilityproject.job.JWTTokenManager;
+import org.dataportabilityproject.job.JobUtils;
+import org.dataportabilityproject.job.PortabilityJob;
+import org.dataportabilityproject.job.TokenManager;
+import org.junit.Before;
+import org.junit.Test;
+
+public class StartCopyHandlerTest {
```

Comment:
Thanks for adding this test!   I'm wondering, if we only ever call validateJobId() for token verification, if that logic could be in a utility class or helper class that we can test directly without bringing in the whole handler.